packages/singleton-manager/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "singleton-manager",
+  "version": "0.0.0",
```

Comment:
What do we do about this? Do we wanna release it straight as 1.0.0 because we can never break it?

Reply:
yes... will do via a commit message with "BREAKING CHANGE" 👍